### PR TITLE
Update ro.m3u

### PR DESCRIPTION
**Pull request by SileC**
add channels

### CODE REVIEW

File: channels/ro.m3u [hunk 1]
```diff
@@ -41,6 +41,8 @@ http://46.4.14.12:9999/btvsvlive/btvsvlive/playlist.m3u8
 http://46.4.14.12:9999/btvsvlive/playlist.m3u8?fluxustv.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Romanian" tvg-logo="https://i.imgur.com/BxUibFh.png" group-title="",Canal33
 https://fms-https1.mediadirect.ro/live3/_definst_/canal33.smil/playlist.m3u8?publisher=83
+#EXTINF:-1  tvg-id="" tvg-name="" tvg-language="Romanian" tvg-logo="https://i.imgur.com/Tyrzwr2.png" group-title="",Columna TV
+http://77.36.61.147/hls/channel4/adaptive.m3u8?fluxustv.m3u8
```
Comment (freearhey) [inline channels/ro.m3u]: Stream is not working.

File: channels/ro.m3u [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#EXTM3U x-tvg-url="http://195.154.221.171/epg/guideromania.xml.gz"
+ tvg-id="" tvg-name="" tvg-language="Romanian" #EXTM3U x-tvg-url="http://195.154.221.171/epg/guideromania.xml.gz"
```
Comment (freearhey) [inline channels/ro.m3u]: Wrong file header

File: channels/ro.m3u [hunk 3]
```diff
@@ -67,10 +69,14 @@ http://edge76.rcs-rds.ro/digi24edge/smil:digi24.smil/playlist.m3u8
 http://46.55.111.242:8080/Rezina.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Romanian" tvg-logo="https://i.imgur.com/xAmYapr.jpg" group-title="",Erdely TV
 http://telekomtv-ro.akamaized.net/shls/LIVE$ErdelyTV/6.m3u8/Level(1677721)?start=LIVE&end=END
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Romanian" tvg-logo="" group-title=""EST TV
+http://89.38.8.130:39435
```
Comment (freearhey) [inline channels/ro.m3u]: There's a comma missing in front of the name of the channel.

File: channels/ro.m3u [hunk 4]
```diff
@@ -67,10 +69,14 @@ http://edge76.rcs-rds.ro/digi24edge/smil:digi24.smil/playlist.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Romanian" tvg-logo="https://i.imgur.com/JK6Rypn.jpg" group-title="",Iasi TV Life
 http://webmobile.xdev.ro:81/tv24/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Romanian" tvg-logo="https://i.imgur.com/JK6Rypn.jpg" group-title="",Iasi TV Life
 http://webmobile.xdev.ro:81/tv24/playlist.m3u8?time=1464028219168
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Romanian" tvg-logo="" group-title="",IMPACT TV
+http://89.36.10.12:8080
```
Comment (freearhey) [inline channels/ro.m3u]: Stream is not working.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please make corrections to this pull request so that we can approve it.

**Comment by SileC** [comment]
I have no idea how to fix from pull. They are only day programs in Romania. Just close the pull and I will check and resubmit

**Comment by SileC** [comment]
Will try again.

**Comment by freearhey** [comment]
There is a button to edit the pull request:
Captura de pantalla 2019-12-21 a las 16 21 55

**Comment by SileC** [comment]
I just closed the wrong pull requests and submitted new ones. Hopefully OK this time.

**Pull request closed without merging**